packages/core/src/dialog/DialogHeader.tsx

```diff
@@ -30,15 +35,25 @@ export interface DialogHeaderProps extends ComponentPropsWithoutRef<"div"> {
    * Displays the preheader just above the header
    **/
   preheader?: ReactNode;
+  /**
+   * Description text is displayed just below the header
+   **/
+  description?: ReactNode;
```

Comment:
This is new feature, worth call out, i think there's a ticket to this, asked by pepper?